### PR TITLE
[rayservice] Remove dagdriver from ray_v1alpha1_rayservice.yaml

### PR DESCRIPTION
**Pull request by zcin**
## Checks

- [ ] I've made sure the tests are passing. 
- Testing Strategy
   - [ ] Unit tests
   - [ ] Manual tests
   - [ ] This PR is not tested :(

### CODE REVIEW

**Comment by kevin85421** [comment]
Would you mind providing more details about the reasons behind this change? In particular, I am wondering what will happen if users deploy this RayService with an older version of Ray, such as Ray 2.7.0. Will it fail?

**Comment by zcin** [comment]
> Would you mind providing more details about the reasons behind this change? In particular, I am wondering what will happen if users deploy this RayService with an older version of Ray, such as Ray 2.7.0. Will it fail?

@kevin85421 No, it won't fail. This is simply changing the application code to not use DAGDriver. There is nothing backwards incompatible with this change. The only thing that will fail is if you try to use DAGDriver in Ray 2.9+, which is the reason I need to make this change.

**Comment by kevin85421** [comment]
Would you mind checking the CI failures? I have already retried the tests 3 times. Thanks!

File: tests/test_sample_rayservice_yamls.py [hunk 1]
```diff
@@ -190,7 +190,7 @@ def set_up_cluster(self):
         self.rayservice_name = self.cr["metadata"]["name"]
         self.default_queries = [
             {"path": "/fruit", "json_args": ["MANGO", 2], "expected_output": "6"},
-            {"path": "/calc", "json_args": ["MUL", 3], "expected_output": '"15 pizzas please!"'},
+            {"path": "/calc", "json_args": ["MUL", 3], "expected_output": '15 pizzas please!'},
```
Comment (kevin85421) [inline tests/test_sample_rayservice_yamls.py:193]: Can we use `"` instead of `'` to keep the consistency?

**Comment by zcin** [comment]
@kevin85421 I have fixed the tests

**Review by kevin85421 — COMMENTED** [review]
* Would you mind explaining why do we need to change from `'"...:'` to `'....'`?
* Do we need to update docs like https://docs.ray.io/en/latest/cluster/kubernetes/user-guides/rayservice.html#kuberay-rayservice and https://docs.ray.io/en/latest/cluster/kubernetes/getting-started/rayservice-quick-start.html#kuberay-rayservice-quickstart?

Thanks!

**Comment by zcin** [comment]
> * Would you mind explaining why do we need to change from `'"...:'` to `'....'`?

DAGdriver converts all output into json format. Removing DAGdriver from the application means the text remains `hi` instead of `"hi"`.

> * Do we need to update docs like https://docs.ray.io/en/latest/cluster/kubernetes/user-guides/rayservice.html#kuberay-rayservice and https://docs.ray.io/en/latest/cluster/kubernetes/getting-started/rayservice-quick-start.html#kuberay-rayservice-quickstart?

Yes, I have updated them in https://github.com/ray-project/ray/pull/41162

**Review by kevin85421 — APPROVED** [review]

**Pull request merged**